src/QuickMarcEditor/QuickMarcEditorRows/FixedField/HoldingsFixedField.js

```diff
@@ -37,7 +37,7 @@ const config = {
     {
       name: 'Copies',
       type: SUBFIELD_TYPES.STRING,
-      length: 4,
+      length: 3,
```

Comment:
What is `Copies` in https://www.loc.gov/marc/bibliographic/bd008.html?

Reply:
This is a Holdings 008 field, it has different properties
https://www.loc.gov/marc/holdings/hd008.html